### PR TITLE
fix(component/sidebarnav): alter the class name for create channel list

### PR DESCRIPTION
**Pull request by sriramanvellingiri**
sidebarnav

### CODE REVIEW

File: react/src/components/blocks/ChatWindow/ChatHeader/ChannelMembers/ChannelMembers.js [hunk 1]
```diff
@@ -6,20 +6,8 @@ import userIconImg from '../../../../../static/images/usericon.png'
 function ChannelMembers({ members = [] }) {
   return (
     <div className={styles.channelMembers}>
```
Comment (curioustushar) [inline react/src/components/blocks/ChatWindow/ChatHeader/ChannelMembers/ChannelMembers.js:9]: @sriramanvellingiri Any specific reason for switching to a hardcoded users list over the existing dynamic user list (which changes according to channel)?

Comment (sriramanvellingiri) [reply react/src/components/blocks/ChatWindow/ChatHeader/ChannelMembers/ChannelMembers.js:9]: @curioustushar the rendering logic code is updated now. checked with real stack, if the member count is 1 the image icon doesnt appear and if the count is more, only two image icon will be appearing,

Comment (curioustushar) [reply react/src/components/blocks/ChatWindow/ChatHeader/ChannelMembers/ChannelMembers.js:9]: @sriramanvellingiri LGTM.

File: react/src/components/blocks/ChatWindow/ChatHeader/ChannelMembers/ChannelMembers.js [hunk 2]
```diff
@@ -6,20 +6,25 @@ import userIconImg from '../../../../../static/images/usericon.png'
...
-      {members &&
+      <img src={userImg} alt="user1" width="100%" height="100%" className={styles.memberImg}/>
+      
+      {members && members.length >1 &&
         members.map((member, index) => {
-          return (
-            <img
-              key={index}
-              src={index % 2 ? userImg : userIconImg}
```
Comment (curioustushar) [inline react/src/components/blocks/ChatWindow/ChatHeader/ChannelMembers/ChannelMembers.js:14]: @sriramanvellingiri Can we keep the logic to have two between different images?

Comment (sriramanvellingiri) [reply react/src/components/blocks/ChatWindow/ChatHeader/ChannelMembers/ChannelMembers.js:14]: @curioustushar two different images are added

Comment (curioustushar) [reply react/src/components/blocks/ChatWindow/ChatHeader/ChannelMembers/ChannelMembers.js:14]: LGTM

**Pull request merged**